File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = get_int("Height:");

    while (height >8 || height <1)
    {
        printf("Again!\n");
        height=get_int("Height:");
    }


    for(int row = 1; row < height + 1; row = row + 1)
    {
        int numSpaces = height - row;
        for (int count = 0; count < numSpaces; count = count + 1)
        {
            printf(" ");
        }

        int numHex = height - numSpaces;
        for(int count = 0;count < numHex; count = count + 1)
        {
            printf("#");
        }
        
        printf(" ");
        
        numHex = height - numSpaces;
        for(int count = 0;count < numHex; count = count + 1)
        {
            printf("#");
        }
        
        numSpaces = height - row;
        for (int count = 0; count < numSpaces; count = count + 1)
        {
            printf(" ");
        }

        printf("\n");
        
    }
}

File: mario.py
from cs50 import get_int

def main():


    height = get_int("Height: ")

    while height > 8 or height < 1:
        height = get_int("Height: ")
        
    mario(height)
    
def mario(height):    
    
    row = 1  
    
    #left side
    for i in range(height):
        space = height - row
        print((" " * space)+("#" * row)+(" ")+("#" * row))
        row = row + 1 
main()